conmon/conmon.c

```diff
@@ -296,6 +296,27 @@ const char *stdpipe_name(stdpipe_t pipe)
 	}
 }
 
+/*
+* reopen_log_file reopens the log file fd.
+*/
+static void reopen_log_file(void)
+{
+	_cleanup_free_ char *opt_log_path_tmp = g_strdup_printf("%s.tmp", opt_log_path);
+
+	/* Close the current log_fd */
+	close(log_fd);
+
+	/* Open the log path file again */
+	log_fd = open(opt_log_path_tmp, O_WRONLY | O_TRUNC | O_CREAT | O_CLOEXEC, 0600);
+	if (log_fd < 0)
+		pexit("Failed to open log file %s: %s", opt_log_path, strerror(errno));
```

Comment:
With #1368 landed, this should be updated to `pexitf`. and since `pexit*` includes `strerror` on its own, you just need:

```C
pexitf("Failed to open log file %s", opt_log_path);
```